@here/harp-mapview/lib/Utils.ts

```diff
@@ -239,9 +239,143 @@ export namespace MapViewUtils {
             groundPlane.constant = 0;
             groundSphere.radius = EarthConstants.EQUATORIAL_RADIUS;
         }
+        if (result !== null) {
+            return result;
+        }
+        return getGazingTargetFromCamera(camera, projection, elevation);
+    }
+
+    function getGazingTargetFromCamera(
+        camera: THREE.Camera,
+        projection: Projection,
+        elevation?: number
+    ): THREE.Vector3 | null {
+        const cameraPos = cache.vector3[0].copy(camera.position);
+        const cameraLookAt = camera.getWorldDirection(cache.vector3[1]).normalize();
+        let result: THREE.Vector3 | null = null;
+        // Find gazing point on the ground
+        if (projection.type === ProjectionType.Planar) {
+            // Tolerance may be adjusted to not exactly hit the horizon line.
+            //const planarTolerance = THREE.MathUtils.degToRad(1.0);
+            if (camera instanceof THREE.PerspectiveCamera) {
```

Comment:
Please remove and expect `PerspecitveCamera` as parameter.
It looks like we are supporting other cameras but we dont't.

Reply:
Hehe, so you don't believe in Ortho mode :).